app/templates/@layout.latte

```diff
@@ -63,6 +63,7 @@
 		{block main}
 		<div class="main" lang="en">
 			<div class="content has-sidebar">
+				<div class="flash error">See our new modern addons portal <a href="http://bit.ly/cmtte">componette.com</a>. Completely automatic.</div>
```

Comment:
My point was not about the tool itself, but that you redirect of HTTP which is completely unnecesary and is in fact ilogical, when all three websites are on https.